src/sdk/namespace_cache.js

```diff
@@ -561,6 +568,16 @@ class NamespaceCache {
 
             if (cache_ok) {
                 assert.strictEqual(hub_res.value.etag, cache_res.value.etag);
+
+                const update_params = _.pick(_.defaults(
+                    { bucket: this.namespace_nb.target_bucket }, params), 'bucket', 'key', 'last_modified_time');
```

Comment:
Need to match obj_id as well here, to avoid overriding another object with the same key (concurrent overwrites).